src/rustup-cli/common.rs

```diff
@@ -286,13 +287,27 @@ pub fn list_overrides(cfg: &Cfg) -> Result<()> {
     let overrides = try!(cfg.settings_file.with(|s| Ok(s.overrides.clone())));
 
     if overrides.is_empty() {
-        println!("no overrides");
+        println!("No overrides.");
     } else {
+        println!("Overrides:");
+        let mut any_not_exist = false;
```

Comment:
I'd prefer not to make these UI changes at this time, to keep some consistency with `toolchain list`, and because there are already a variety of inconsistencies and issues with the UI that I'd prefer to tackle more holistically later. Can you keep the empty message "no overrides"; remove the "Overrides:" header; add a single blank `println!("")` before the "you may remove overrides ..." hint?